sympy/functions/combinatorial/tests/test_comb_factorials.py

```diff
@@ -497,6 +499,9 @@ def test_binomial_Mod_slow():
     assert Mod(binomial(9734, 451, evaluate=False), q) == Mod(binomial(9734, 451), q)
     assert Mod(binomial(-10733, 4459, evaluate=False), q) == Mod(binomial(-10733, 4459), q)
     assert Mod(binomial(-15733, 4458, evaluate=False), q) == Mod(binomial(-15733, 4458), q)
+    assert Mod(binomial(23, 38), q) == S.Zero
+    assert Mod(binomial(4556, 38635874), q) == S.Zero
+    assert Mod(binomial(456, 789), q) == S.Zero
```

Comment:
These still doesn't increase the coverage for the diff for now.
What about changing these to
```python3
    assert Mod(binomial(23, -38, evaluate=False), q) is S.Zero
    assert Mod(binomial(23, 38, evaluate=False), q) is S.Zero
```